dev-docs/bidders/mediaConsortium.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: bidder
+title: Media Consortium
+description: Prebid Media Consortium Bidder Adapter
+biddercode: mediaConsortium
+media_types: banner, video
+pbjs: true
+userId: all
+multiformat_supported: will-bid-on-any
+sidebarType: 1
+---
+
+### Prebid JS
+
+To get acces to the full feature set of the adapter you'll need to allow localstorage access in the `bidderSettings`.
+
+```javascript
+    pbjs.bidderSettings = {
+        mediaConsortium: {
+            storageAllowed: true
+        }
+    }
+```
+
+#### Controlling 1plusX profile API usage and FPID retrieval
+
+You can use the `setBidderConfig` function to enable or disable 1plusX profile API usage and fpid retrieval.
+
+If the keys found below are not defined, their values will default to `false`.
+
+```javascript
+    pbjs.setBidderConfig({
+        bidders: ['mediaConsortium'],
+        config: {
+            // Controls the 1plusX profile API usage
+            useProfileApi: true,
+            // Controls the 1plusX fpid retrieval
+            readOnePlusXId: true
+        }
+    });
+```
+
+#### Required video parameters
```

Comment:
what about banner parameters?

Reply:
I've updated the title to better reflect the content of this section, let me know if it is not enough